.github/workflows/build-artifact.yml

```diff
@@ -0,0 +1,69 @@
+name: Build artifact
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  detect-files-changed:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+      - name: Test files changed
+        id: files-changed
+        uses: tj-actions/changed-files@v44
+        with:
+          fetch_additional_submodule_history: 'true'
+          files: |
+            tests/ansibench/*
+            tests/rv8-bench/*
+            tests/*.c
```

Comment:
May I know if we add a directory of test suite, can these syntax detect a new directory and any subdirectories of it? E.g., `tests/system`

Reply:
If the new directory is added, this rule should be updated too.

Reply:
My initial intention is to ensure that if the system emulation is merged in the future, additional test suites will be included as well. To distinguish these from the ISA simulation test suite, I'd like to add a new directory (e.g., `tests/system`).

Thus, we shall support detection of a new test suite directory and any subdirectories of it.

Reply:
From the usage [section](https://github.com/tj-actions/changed-files?tab=readme-ov-file#usage-),

> Matching Files and Folders: To match all files and folders under a directory, this requires a globstar pattern e.g. dir_name/** which matches any number of subdirectories and files.

I think updating the rules with `tests/system/**` is fine.

Reply:
> From the usage [section](https://github.com/tj-actions/changed-files?tab=readme-ov-file#usage-),
> 
> > Matching Files and Folders: To match all files and folders under a directory, this requires a globstar pattern e.g. dir_name/** which matches any number of subdirectories and files.
> 
> I think updating the rules with `tests/system/**` is fine.

Thanks for the information. I think it is not appropriate to change the rule in this PR. When system emulation are integrated, I will consider the change.